apps/common/controllers/address/helper.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
+
+const PostcodeController = require('./postcode');
+const SelectAddressController = require('./select');
+const ManualAddressController = require('./manual');
+
+function applyDefaults(config) {
+  return config;
+}
+
+function validateConfig(config) {
+  const required = [
+    'prefix',
+    'start',
+    'select',
+    'manual',
+    'next'
+  ];
+
+  required.forEach((key) => {
+    if (!config[key]) {
+      throw new Error(`options.${key} must be defined`);
```

Comment:
I'm going to take the fact that you're arguing the semantics of "config" vs "options" in error messaging as a good sign.

> I wanted to get a review of the overall ideas, before going too far into the detail